code/modules/client/preference_setup/controls/01_keybindings.dm

```diff
@@ -23,7 +24,7 @@
 		if(length(user_binds[kb.name]))
 			continue // key is unbound and or bound to something
 		var/addedbind = FALSE
-		if(hotkeys)
+		if(client.prefs?.hotkeys)
```

Comment:
Why?

Reply:
Idk I running on this place twice and probably mass-replaced this.